playbooks/keyholders/EXISTING_SIGNER.md

```diff
@@ -101,9 +101,10 @@ This will setup your clone and build the TUF binary to use for metadata generati
 
 ### Signing
 
-When prompted in the Slack channel, begin signing root and targets metadata. Run:
+When prompted in the Slack channel, begin signing root and targets metadata. If the ceremony started on a prior date, you should add a repository reference by setting `REPO`. Otherwise, for same-day signing, you may omit to default to the current date. Run:
 ```
-./scripts/step-2.sh
+REPO=$(pwd)/ceremony/$DATE ./scripts/step-2.sh
```

Comment:
```suggestion
REPO=$(pwd)/ceremony/<START_DATE> ./scripts/step-2.sh
```
This suggestion, I think, makes it clearer that this is a user-supplied value (not an expansion like `$(pwd)`.